frontend/app/scripts/app.js

```diff
@@ -29,7 +29,27 @@ angular
         controller: 'AboutCtrl',
         controllerAs: 'about'
       })
+      .when('/userprofile', {
+        templateUrl: 'views/userprofile.html',
+        controller: 'UserProfileCtrl',
+        controllerAs: 'userprofile'
+      })
       .otherwise({
         redirectTo: '/'
       });
-  });
+  }).run(function ($rootScope) {
+    //TODO provide role from userService here
+    $rootScope.isAdmin = function () {
```

Comment:
They not rootScope method. So rootScope contained date for user. Until we didn't add user service, add user json data manually. And move this method to Header controller.